packages/ses/src/lockdown.js

```diff
@@ -115,21 +137,12 @@ const assertDirectEvalAvailable = () => {
     // and indirect, which generally creates a new global.
     // We are going to throw an exception for failing to initialize SES, but
     // good neighbors clean up.
-    if (!allowed) {
+    if (!directEvalAllowed) {
       delete globalThis.SES_changed;
```

Comment:
What's `SES_changed`?

Reply:
the global var created as a side effect when the host (Hermes) only has indirect eval available (same as [before](https://github.com/endojs/endo/blob/f27ab4d422d7e367036c2d75c27e2eea97db361c/packages/ses/src/lockdown.js#L103-L133))